bokehjs/test/size.coffee

```diff
@@ -10,8 +10,8 @@ LIMITS = {
   "css/bokeh-widgets.min.css": 527
   "css/bokeh.min.css":         74
   "js/bokeh-widgets.min.js":   290
-  "js/bokeh-api.min.js":        40
-  "js/bokeh.min.js":           918
+  "js/bokeh-api.min.js":        70
+  "js/bokeh.min.js":           780
```

Comment:
Currently `bokeh.min.js` is at 757 kB and I'm still working on webgl split-off, which will result in at least -60 kB.